CHANGELOG.md

```diff
@@ -43,10 +45,11 @@ Here is a template for new release sections
 - Modified the doc-strings in the definitions of some functions to abide by the formatting rules of numpy doc-strings (#505)
 - Suppressed the log messages of the Flask server (for report webapp) (#509) 
 - Move bulk data preparation code for report from F2 into E1 and F1 modules and into functions
- (#511)
+ (#511, #512)
+ - F2 now calls functions from F1 to prepare the figures of the report (#512)
 
 ### Removed
--
+- Functions to generate plots with matplotlib in F1 (#512)
 
```

Comment:
```suggestion
- Many tests that checked if matplot lib plots were stored to file, not replaced by new tests for storing plotly graphs to file (#512)
```

Reply:
I didn't rewrite the tests because the file will always be created with plotly, it will just be a white image if anything didn't work internally. Thus the tests of looking for file existence is not very complete here.